tools/tagger/index.html

```diff
@@ -179,7 +179,7 @@
         selected.forEach((e) => {
             tagInfo=fetchTag(e);
             html+=`<span data-target="${e}" onclick="remove(this)" class="path active">${tagInfo.path}<span class="tag">${tagInfo.tag}</span></span> `;
-            if (tagInfo.path.toLowerCase().indexOf("products")==0) products+=tagInfo.tag+", ";
+            if (productPaths.includes(tagInfo.path.trim())) products+=tagInfo.tag+", ";
```

Comment:
You know what you are doing, but the code is doing something completely different and looks weird: why a path would only be CC or EC... ?

Reply:
Should be working fine for the moment, but I'll add safeguarding